cegs_portal/templates/account/logout.html

```diff
@@ -1,19 +1,73 @@
-{% extends "account/base.html" %}
+{% extends "account/base.html" %} {% load i18n %} {% block css %} {{ block.super }}
 
-{% load i18n %}
+<style>
+    .container {
+        background-color: white;
+        padding: 20px;
+        border-radius: 10px;
+        box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+        font-family: "Arial", sans-serif;
+        color: #094474;
+        max-width: 800px;
+        margin: 0 auto;
+        text-align: center;
+    }
 
-{% block head_title %}{% translate "Sign Out" %}{% endblock %}
+    .cancel-sign-out-button,
+    .sign-out-button {
+        display: inline-block;
+        padding: 0.5rem 1rem;
+        font-weight: bold;
+        border-radius: 0.25rem;
+        margin-top: 10px;
+    }
 
-{% block inner %}
-<h1>{% translate "Sign Out" %}</h1>
+    .cancel-sign-out-button {
+        background-color: rgb(82, 116, 210);
+        color: #ffffff;
+        text-decoration: none;
+    }
 
-<p>{% translate 'Are you sure you want to sign out?' %}</p>
+    .cancel-sign-out-button:hover {
+        color: #ffffff;
+        background-color: rgb(29, 78, 216);
+    }
+
+    .sign-out-button {
+        background-color: rgb(156, 163, 175);
+        color: #ffffff;
+    }
+
+    .sign-out-button:hover {
+        background-color: #c02626;
+    }
+
+    h1 {
+        font-size: 1.25rem;
+        font-weight: bold;
+        line-height: 1.75rem;
+    }
+
+    .faded-line {
```

Comment:
This class isn't used, I think you should remove it.

Reply:
Thanks for the compliment, Thomas. Fixed the "No" button and removed the unused class.

Reply:
![image](https://user-images.githubusercontent.com/19657615/233449760-e7fbc709-a2d8-4ee1-9953-e0d759194752.png)